Runtime/Native/Windows/NativeClient.cs

```diff
@@ -0,0 +1,438 @@
+                attachments.Count());
+
+            if (!_captureNativeCrashes)
+            {
+                Debug.LogWarning("Backtrace native integration status: Cannot initialize Crashpad client");
+                return;
+            }
+
+            foreach (var attribute in clientAttributes)
+            {
+                AddNativeAttribute(attribute.Key, attribute.Value);
+            }
+
+            // add exception type to crashes handled by crashpad - all exception handled by crashpad 
+            // by default we setting this option here, to set error.type when unexpected crash happen (so attribute will present)
+            // otherwise in other methods - ANR detection, OOM handler, we're overriding it and setting it back to "crash"
+
+            // warning 
+            // don't add attributes that can change over the time to initialization method attributes. Crashpad will prevent from 
+            // overriding them on game runtime. ANRs/OOMs methods can override error.type attribute, so we shouldn't pass error.type 
+            // attribute via attributes parameters.
+            AddNativeAttribute("error.type", "Crash");
+        }
+
+        public void Disable()
+        {
+            if (_anrThread != null)
+            {
+                _stopAnr = true;
+            }
+            return;
+        }
+
+        public void GetAttributes(IDictionary<string, string> attributes)
+        {
+            return;
+        }
+
+        public void HandleAnr(string gameObjectName = "", string callbackName = "")
+        {
+            if (!_captureNativeCrashes || !_configuration.HandleANR)
+            {
+                return;
+            }
+
+            bool reported = false;
+            var mainThreadId = Thread.CurrentThread.ManagedThreadId;
+            _anrThread = new Thread(() =>
+            {
+                float lastUpdatedCache = 0;
+                while (_anrThread.IsAlive && _stopAnr == false)
+                {
+                    if (!_preventAnr)
+                    {
+                        if (lastUpdatedCache == 0)
+                        {
+                            lastUpdatedCache = _lastUpdateTime;
+                        }
+                        else if (lastUpdatedCache == _lastUpdateTime)
+                        {
+                            if (!reported)
+                            {
+
+                                reported = true;
+                                if (AndroidJNI.AttachCurrentThread() == 0)
+                                {
+                                    // set temporary attribute to "Hang"
+                                    AddNativeAttribute("error.type", "Hang");
+
+                                    NativeReport("ANRException: Blocked thread detected.", true);
+                                    // update error.type attribute in case when crash happen 
+                                    AddNativeAttribute("error.type", "Crash");
```

Comment:
Is it possible that another crash might occur in another thread before we can set `error.type` back to `Crash`?

Reply:
yes, there is a small chance that this migh happen

Reply:
Maybe it's worth it to add a comment to warn users of this potentially unexpected behavior?